docker-compose.dev.yml

```
@@ -20,15 +20,13 @@ services:
 
   spot:
     # NOTE: This will be replaced by https://github.com/samply/spot soon.
-    image: docker.verbis.dkfz.de/ccp-private/central-spot
+    image: samply/rustyspot:refactor-evenMore
```

This should be main too right?
```suggestion
    image: samply/rustyspot:main
```

---

change applied